extensions/amp-ad/amp-ad.md

```diff
@@ -182,33 +184,38 @@ This is due to the UX implications of full page overlay ads. It may be considere
 
 ## Attributes
 
-### type
+**type**
```

Comment:
OK, agree that bold is probably better than titles. Fine for now, but looking forward to a saner future solution.